tests/test_migrate_firefox_profile.py

```diff
@@ -12,6 +12,74 @@
 emfp = import_script_as_module("emfp", system_script("eos-migrate-firefox-profile"))
 
 
+class TestUpdateMimeappsList(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+
+        self.tmp = tempfile.TemporaryDirectory()
+
+    def tearDown(self):
+        self.tmp.cleanup()
+
+    def test_nonexistent(self):
+        emfp.update_mimeapps_list(os.path.join(self.tmp.name, "mimeapps.list"))
+
+    def test_not_there(self):
+        orig_data = textwrap.dedent(
+            """
+            [Default Applications]
+            image/jpeg=eog.desktop
+            """
+        ).lstrip()
+        expected_data = orig_data
+
+        self._test(orig_data, expected_data)
+
+    def test_there(self):
+        orig_data = textwrap.dedent(
+            """
+            [Default Applications]
+            text/html=org.mozilla.Firefox.desktop
```

Comment:
I'm sure GKeyFile will do the right thing, but might be good to make the test data have multiple entries in the list to make sure the correct entry is replaced and the order is maintained.

Reply:
Good point. I added an `[Added Applications]` section which is where, in practice, there are plural entries.